db/seeds.d/07-provisioning_templates.rb

```diff
@@ -105,7 +17,8 @@
 
       t = ProvisioningTemplate.create({
         :snippet  => false,
-        :template => contents
+        :template => contents,
+        :locked => true
       }.merge(input))
 
       if t.default?
```

Comment:
yes, it's always set like that few lines above, I'll remove the if condition while I'm at it